client/common/directives/ModelFieldImage/ModelFieldImage.js

```diff
@@ -25,7 +25,7 @@ angular.module('dashboard.directives.ModelFieldImage', [
   return {
     restrict: 'E',
     require: '^form',
-    template: '<div class="image-container" style="background: no-repeat center center url(\'{{ thumbnailUrl }}\'); background-size: contain;" ng-click="imageClick()"></div> \
+    template: '<div class="image-container" ng-class="{ \'display-only\' : displayOnly }" style="background: no-repeat center center url(\'{{ thumbnailUrl }}\'); background-size: contain;" ng-click="!displayOnly && imageClick()"></div> \
```

Comment:
You can wrap the image-container class as part of the ng-class as well. Instead of relying on CSS priority you can explicitly indicate a class.

Reply:
Done.